metabox/metabox/metabox-provider/bin/dependency_test.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# Copyright (C) 2023 Canonical Ltd.
+#
+# Authors:
+#   Massimiliano Girardi <massimiliano.girardi@canonical.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3,
+# as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+This module imports all dependencies of checkbox-ng and checkbox-support.
```

Comment:
This was done this way because I did not want to create a weird machinery to read the pyproject file and extract the dependencies at runtime. That would be possibly harder to read although possible for sure. The manually updating the file part is due to this design, if you want I can make it automatic but it may be more trouble than it is worth

Reply:
Reporting Meet discussion: We agreed that the situation is suboptimal, but making this code clean and orderly is well outside the scope ot this pr and very hard, so this will have to suffice for the time being